README.md

```diff
@@ -8,6 +8,53 @@ J. H. Legarreta, M. Descoteaux, and P.-M. Jodoin. “PROCESSING OF TRACTOGRAPHY
 RESULTS USING AN AUTOENCODER”. Filed 03 2021. Imeka Solutions Inc. United States
 Patent #17/337,413. Pending.
 
+## Installation
+
+To use tractolearn, it is recommended to create a virtual environment using python 3.8 that
+will host the necessary dependencies:
+
+```sh
+   virtualenv tractolearn_env --python=python3.8
+   source tractolearn_env/bin/activate
+```
+
+They can be installed by executing:
+
+```sh
+   pip install -r requirements.txt
+   pip install -r requirements_dmri.txt
```

Comment:
This has been deprecated: https://peps.python.org/pep-0518/

The above and some other related PEPs have introduced a more modern/robust way to specify the dependencies. I will come up with a new PR once this one gets merged to modernize that. In fact, the CI workflow file does already use such a modern convention.

Reply:
Do you want me to do something about it ? You want to use the `pyproject.toml` approach I guess

Reply:
We will not be able to test that the PR is passing if we do not transition to it. But maybe it can be done in a future PR.

Reply:
Ok good

Reply:
Done